### PR TITLE
ci: ensure gas snapshots are updated

### PR DESCRIPTION
**Pull request by 0xteddybear**

### CODE REVIEW

**Comment by 0xteddybear** [comment]
> would you modify the snapshot just to check the result? 🙏

I did! And I'm not getting any errors :upside_down_face: I'm troubleshooting that

File: .forge-snapshots/swapExactAmountIn.snap [hunk 1]
```diff
@@ -1 +1 @@
-82842
\ No newline at end of file
+105194
```
Comment (wei3erHase) [inline .forge-snapshots/swapExactAmountIn.snap:1]: any clue when did this happen??

Comment (wei3erHase) [reply .forge-snapshots/swapExactAmountIn.snap:1]: surprisingly, it happened in #110, i don't fully understand how, even the Pool snaps suffered an increase 🤯

**Review by wei3erHase — APPROVED** [review]
good job! 👍

**Pull request merged**